content/articles/2019-08-gargle-hello-world.Rmarkdown

```diff
@@ -0,0 +1,88 @@
+---
+title: gargle's debut on CRAN
+slug: gargle-hello-world
+author: Jenny Bryan
+description: >
+  gargle is now on CRAN.
+date: '2019-08-21'
+categories: [package]
+tags:
+  - gargle
+photo:
+  url: https://flic.kr/p/oe2LJ1
+  author: North Carolina Christian advocate (1894)
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+## Introduction
+
+We're elated to announce the initial release of gargle and of googledrive v1.0.0 on CRAN!
+
+  * The [gargle package](https://gargle.r-lib.org) is meant to take some of the pain out of working with Google APIs. It's mostly aimed at the maintainers of R packages that call Google APIs. If we're successful, users won't really notice gargle, they'll just feel like these packages got easier to work with and less idiosyncratic.
+  * The [googledrive package](https://googledrive.tidyverse.org) is a great example of such a package: it helps useRs work with Google Drive from R. The v1.0.0 release gets all auth functionality (and more) from gargle and also has a new feature that makes file upload/update more user-friendly.
+  
+This post focuses on gargle, but don't forget to check out the companion post describing googledrive v1.0.0. The bigrquery package also uses gargle as of v1.2.0, which you can read about in its [changelog](https://bigrquery.r-dbi.org/news/index.html#bigrquery-1-2-0).
```

Comment:
Maybe this is coming, but it’d be great to have a link to the companion post where you write _don’t forget to check out the companion post_

Reply:
Done. On both sides.